CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
-## [12.2.3](https://github.com/thymikee/jest-preset-angular/compare/v12.2.2...v12.2.3) (2022-11-26)
+## [13.0.0](https://github.com/thymikee/jest-preset-angular/compare/v13.2.3...v13.0.0) (2023-01-07)
```

Comment:
You don't need to update changelog because it is generated via conventional changelog later